portfolio.html

```diff
@@ -0,0 +1,96 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+	<title>Max G Portfolio</title>
+	<link rel="Stylesheet" type="text/css" href="portfolio.css">
```

Comment:
you've missed the reset.css (normalize) file. This means the site will appear differently on different browsers as they have different defaults for things you having specified. Let me know if you need more info about this.